src/middleware.ts

```diff
@@ -199,4 +281,6 @@ export {
   defaultMiddleware,
   defaultServerMiddlewareWrapper,
   defaultClientMiddlewareWrapper,
+  timeoutMiddlewareClient,
+  timeoutMiddlewareServer,
```

Comment:
Are these 2 the only additional middleware? Also don't we only need to export the default middleware? What other middleware aren't we exporting?

Reply:
i'm exporting these for the sake of tests, we're also exporting all other middleware from this file